source/Octopus.Client/Repositories/Async/IMixScopeRepository.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Octopus.Client.Repositories.Async
+{
+    public interface IMixScopeRepository<TResource>
```

Comment:
Should be "MixedScope". Also, I don't think we want to expose this abstraction to consumers of the client. It is difficult enough to explain the concept of mixed-scope endpoints internally, let alone to our customers.